f2/src/forms/EvidenceInfoForm.js

```diff
@@ -176,6 +173,21 @@ export const EvidenceInfoForm = (props) => {
                       <Trans id="evidencePage.addFileButton" />
                     </FileUpload>
                   </Box>
+                  <Text fontSize="md" as="p" fontWeight="bold">
+                    <Trans id="evidencePage.supportedFiles" />
+                  </Text>
+                  <br />
+                  <Ul>
+                    <Li fontSize="md" mb={2}>
+                      <Trans id="evidencePage.fileTypes1" />
+                    </Li>
+                    <Li fontSize="md" mb={2}>
+                      <Trans id="evidencePage.fileTypes2" />
+                    </Li>
+                    <Li fontSize="md" mb={2}>
+                      <Trans id="evidencePage.fileTypes3" />
+                    </Li>
+                  </Ul>
```

Comment:
To bring spacing even better, you can group the list in a box! 
![Capture d’écran, le 2020-05-05 à 11 32 53](https://user-images.githubusercontent.com/5230720/81085749-9f44a900-8ec5-11ea-9ac7-96fa423ef690.png)
```jsx
<Box mb={4}>
  <P fontSize="md" fontWeight="bold">
    <Trans id="evidencePage.supportedFiles" />
  </P>
  <Ul>
    <Li fontSize="md" mb={2}>
      <Trans id="evidencePage.fileTypes1" />
    </Li>
    <Li fontSize="md" mb={2}>
      <Trans id="evidencePage.fileTypes2" />
    </Li>
    <Li fontSize="md" mb={2}>
      <Trans id="evidencePage.fileTypes3" />
    </Li>
  </Ul>
</Box>
````
To get this: 
![Capture d’écran, le 2020-05-05 à 11 44 49](https://user-images.githubusercontent.com/5230720/81085935-db780980-8ec5-11ea-8cc3-209ac7a263bd.png)

Reply:
To be honest with you, I can't tell the difference from the view. but make sense to group them together.